static/css/devhub/new-landing/sections/my-addons.less

```diff
@@ -1,3 +1,150 @@
 .DevHub-MyAddons-copy {
   max-width: 32em;
 }
+
+.DevHub-MyAddons-item {
+  display: flex;
+  align-items: center;
+  border-bottom: 1px solid #e4e4e4;
+  padding-bottom: 25px;
+}
+
+.DevHub-MyAddons-item-modified {
+  margin-left: auto;
+  align-self: flex-end;
+}
+
+.DevHub-MyAddons-item-details {
+  margin-left: 25px;
+
+  .DevHub-MyAddons-item-name {
+    font-size: 24px;
+    font-weight: 400;
+  }
+
+  a {
+    margin-left: 5px;
+    font-size: 15px;
+    text-decoration: none;
+    color: @color-button-default;
+
+    &.DevHub-MyAddons-version-status-incomplete,
+    &.DevHub-MyAddons-version-status-deleted,
+    &.DevHub-MyAddons-version-status-rejected,
+    &.DevHub-MyAddons-version-status-disabled,
+    &.DevHub-MyAddons-version-status-none {
+      color: #9b9898;
+    }
+
+    &.DevHub-MyAddons-version-status-nominated {
+      color: #ff9216;
+    }
+
+    &.DevHub-MyAddons-version-status-approved {
+      color: #329902;
+    }
+
+    &.DevHub-MyAddons-version-status-admin-disabled {
+      color: #d92215;
+    }
+  }
+
+  .DevHub-MyAddons-item-channel-listed,
+  .DevHub-MyAddons-item-channel-unlisted {
+    border-radius: 4px;
+    padding: 2px 5px;
+    font-size: 13px;
+    font-weight: 500;
+    color: #fff;
+  }
+
+  .DevHub-MyAddons-item-channel-listed {
+    background-color: #01bdad;
+  }
+
+  .DevHub-MyAddons-item-channel-unlisted {
+    background-color: #7a2f7a;
+  }
+}
+
+.DevHub-MyAddons-item-icon {
+  width: 48px;
+  height: 48px;
+}
+
+.DevHub-MyAddons-item-versions {
+  display: flex;
+  justify-content: space-between;
+}
+
+.DevHub-MyAddons-item-version-details {
+  width: 320px;
+}
+
+.DevHub-MyAddons-item-buttons {
+  margin-top: 20px;
+  display: flex;
+  flex-direction: row;
+  align-items: flex-start;
+  justify-content: flex-start;
+
+  > a:first-child {
+    font-size: 15px;
+    text-decoration: none;
+    color: @color-button-default;
+  }
+
+  .DevHub-MyAddons-item-buttons-submit {
+    margin-left: auto;
+  }
+}
+
+.DevHub-MyAddons-whatsnew-container {
+  border: 8px solid @color-callout-border;
+  background: #fff;
+  display: flex;
+  flex-flow: row;
+  justify-content: space-between;
+  padding: 30px;
+  margin: 30px 0 60px;
+  position: relative;
+  background-image: url("../../../img/developers/new-landing/whatsnew-background.png");
+  background-repeat: no-repeat;
+  background-position: right 10px bottom 10px;
+
+  &::before {
+    content: '';
+    background-image: url("../../../img/developers/new-landing/whatsnew-rocket.png");
+    background-repeat: no-repeat;
+    background-size: contain;
+    position: absolute;
+    width: 100%;
```

Comment:
This is eating clicks since it's over the top of the other links. `pointer-events: none` will probably solve that.